templates/ai/content/llms-101/_params-carousel.html

```diff
@@ -0,0 +1,27 @@
+{% set slides = [
+  { "alt": "LLM Model Parameter Visualization",title": "Model", "description": "Different models are trained on different sets of data, which influences their response.", "image": "/img/ai/carousels/llm-01-model.png"},
+  { "alt": "Max Length Parameter Visualization",title": "Max Length", "description": "Setting a limit to the number of tokens.", "image": "/img/ai/carousels/llm-02-maxlength.png"},
+  { "alt": "Temperature Parameter Visualization",title": "Temperature", "description": "Degree of randomness. Setting a high temperature may lead to more creative responses. Setting a low temperature will result in responses that are more literal.", "image": "/img/ai/carousels/llm-03-temp.png"},
+  { "alt": "Top P Parameter Visualization",title": "Top P (Probability)", "description": "Include only the top tokens whose percentage likelihood adds up to the specified “Top P” (e.g. 15%).", "image": "/img/ai/carousels/llm-04-top-p.png"},
+  { "alt": "Frequency Parameter Visualization",title": "Frequency", "description": "The higher this is set, the more that repetition of tokens present in the context will be penalized in suggestions.", "image": "/img/ai/carousels/llm-05-frequency.png"}
+]
+%}
```

Comment:
Q: Are these `image`s&hellip; _somewhere_?
I didn't see them in this PR or in your branch at https://github.com/skyfallsin/MIECO/tree/mh/32-redesign-nav-based-on-mc-video-8-14/img/ai or in <kbd>git ls-files **/*.png</kbd> or in **main**.